### PR TITLE
Add linter, pom dependencies

### PR DESCRIPTION
**Pull request by roonil-wazlib**

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (texm) [inline portfolio/pom.xml:1]: Should this directory be named `portfolio`?

Comment (roonil-wazlib) [reply portfolio/pom.xml:1]: No, that's just the project root. It has to be there in all pom files according to the documentation. It was like that in the training project as well.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Ignore generated NPM files
+# These are sometimes included but not necessary for this project
+package-lock.json
```
Comment (texm) [inline .gitignore:3]: I would recommend keeping the lock file in version control

Comment (roonil-wazlib) [reply .gitignore]: Sorted

**Review by Phoebe-Khokgawe — APPROVED** [review]

**Review by texm — APPROVED** [review]

**Pull request merged**